plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/messages.properties

```diff
@@ -0,0 +1,4 @@
+FIELD_EDITOR_ANALYTICS_DISCLAIMER=Your use of this plug-in is subject to the Apache 2.0 software license and the <a href="privacy-policy">Google Privacy Policy</a>.
```

Comment:
Why are these all prefixed with FIELD_EDITOR? They feel more general than that. e.g. just ANALYTICS_DISCLAIMER should be fine.